addons/website/static/tests/tours/snippet_popup_and_animations.js

```diff
@@ -36,6 +36,22 @@ const setOnScrollAnim = function () {
     ];
 };
 
+const waitForAnimationDelayBelow = function(element, threshold=0.5, timeout=2000, interval=50) {
+    return new Promise((resolve, reject) => {
+        const start = luxon.DateTime.now().toMillis();
+        function check() {
+            const animationDelay = parseFloat(element.style.animationDelay) || 0;
+            if (animationDelay < threshold) {
+                resolve();
```

Comment:
Just a tiny thing -- We might want to do 
```suggestion
                return resolve();
```

and same for `reject` -> `return reject`, to make sure we end the execution of the function when the promise is settled, otherwise, it might run `check` infinitely?

I think it's fine here as it is, because of the way we are handling the rejection in the tour, but it could be problematic in other cases, for instance, if we call it as `waitForAnimationDelayBelow(el).then(console.log).catch(console.error)`, `check` will be infinitely called, even after the promise resolves/rejects.